### DISCUSSION TITLE
LB3 API Documentation

### DISCUSSION THREAD

**Discussion by david-newton** [loopbackio/loopback-next#8826] [Q&A]
I'm dealing with a legacy project that was created with loopback 3. Recent attempts to access http://apidocs.loopback.io/loopback/ have been met with an error. Is this known and/or will the documentation be accessible again? Alternatively is this documentation hosted anywhere else?

**Comment by raymondfeng** [comment]
See https://loopback.io/doc/en/lb3/

**Reply by david-newton** [reply]
I was more looking for the content that previously existed @ http://apidocs.loopback.io/loopback/.
I believe it had a full list of the available lb3 classes and their methods.
